tf-psa-crypto/tests/suites/test_suite_psa_crypto_entropy.function

```diff
@@ -8,6 +8,10 @@
 #include "entropy_poll.h"
 
 /* Calculating the minimum allowed entropy size in bytes */
+#if !defined(MBEDTLS_ENTROPY_BLOCK_SIZE)
+#define MBEDTLS_ENTROPY_BLOCK_SIZE      MBEDTLS_ENTROPY_MIN_PLATFORM
+#endif
```

Comment:
Actually  `check_config.h` has a check for [this condition](https://github.com/Mbed-TLS/mbedtls/blob/b8457fff9fd4303b2dfd91d961aed5b81db5e8ef/include/mbedtls/check_config.h#L689) so I suspect that the problem I was trying to fix here was something else. Anyway, since I don't remember where it was exactly and, yes, it was also a quick fix, I can just revert this change and see what happens.